engine/resources/net.go

```diff
@@ -38,6 +38,7 @@ import (
 
 	multierr "github.com/hashicorp/go-multierror"
 	errwrap "github.com/pkg/errors"
+
```

Comment:
unrelated nl?

Reply:
`goimports` complains about this otherwise. `goimports -w` added this change and the other one. Both Linux and macOS have this behavior for me. Maybe the version on CI is behind? Afaict CI doesn't complain about missing these newlines.

Reply:
Weird... They break `gofmt`/`goimports` formatting from time to time apparently. Let's leave this as is until our tests start failing please. It didn't happen on my machine either, so maybe OSX on your machine has a different interpretation??

Reply:
```
~/g/s/g/p/mgmt (master|✔) $ test/test-gofmt.sh 
running test/test-gofmt.sh
FAIL: The following golang files are not properly formatted (goimports -l): engine/resources/net.go
lib/cli.go
~/g/s/g/p/mgmt (master|✔) $ go version 
go version go1.10.4 linux/amd64
```

This is on Linux. But I'll revert the newlines, np.

Reply:
```
$ ./test/test-gofmt.sh 
running ./test/test-gofmt.sh
PASS
$ go version
go version go1.10.7 linux/amd64
```

Is it possible your slightly older version has a different way of doing things? This is very strange.